src/compiler/checker.ts

```diff
@@ -6252,8 +6252,12 @@ namespace ts {
             return links.declaredType;
         }
 
+        function isStringOrNoSubstitutionTemplateLiteral(expr: Node): boolean {

```

Comment:
IIRC there's already a `isStringLikeLiteral` function in utilities.ts

Reply:
Thanks for catching this.